model/src/test/kotlin/utils/FileArchiverTest.kt

```diff
@@ -162,5 +165,33 @@ class FileArchiverTest : StringSpec() {
             archiver.unarchive(targetDir, PROVENANCE) shouldBe true
             targetDir shouldContainNFiles 0
         }
+
+        "exclude basic binary license file" {
+            createFile("License").writeBytes(byteArrayOf(0xFF.toByte(), 0xD8.toByte()))
+
+            val archiver = FileArchiver.createDefault()
+
+            archiver.archive(workingDir, PROVENANCE)
+
+            val result = archiver.unarchive(targetDir, PROVENANCE)
```

Comment:
Remove empty lines in between here to maintain the AAA-pattern. Same for the other test.